### DISCUSSION TITLE
-claim function

### DISCUSSION THREAD

**Discussion by mrcsdl** [discord-tickets/bot#104] [General]
Hey,

first of all i love your Bot. It works perfectly for me but i got an feature that would be awsome if you can implement this.

if you could implement a -claim so that one Supporter can claim this Ticket, that would be great

**Comment by wolfier74** [comment]
Good idea

**Comment by eartharoid** [comment]
This has been requested numerous times by others, so I will try to include some sort of ticket claiming feature in the v3 update. :+1:
